config/metar.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+// Config for downloading METARs in the context of MSL and RPS
+return [
+    'vatsim_url' => env('VATSIM_METAR_URL', 'metar.vatsim.net'),
+    'regional_url' => env('APP_REGIONAL_PRESSURES_URL'),
```

Comment:
Would it be sensible to provide a default?

Reply:
Done!